spec/index.bs

```diff
@@ -4695,10 +4701,15 @@ Issue: convert "Valid Texture Copy Range" into an algorithm with parameters, sim
                 - [$validating GPUBufferCopyView$](|source|) returns `true`.
                 - |source|.{{GPUBufferCopyView/buffer}}.{{GPUBuffer/[[usage]]}} contains
                     {{GPUBufferUsage/COPY_SRC}}.
-                - [$validating GPUTextureCopyView$](|destination|) returns `true`.
+                - [$validating GPUTextureCopyView$](|destination|, |copySize|) returns `true`.
                 - |destination|.{{GPUTextureCopyView/texture}}.{{GPUTexture/[[textureUsage]]}} contains
                     {{GPUTextureUsage/COPY_DST}}.
                 - |destination|.{{GPUTextureCopyView/texture}}.{{GPUTexture/[[sampleCount]]}} is 1.
+                - If |destination|.{{GPUTextureCopyView/texture}}.{{GPUTexture/[[format]]}} is a depth-stencil format:
+                    - |destination|.{{GPUTextureCopyView/aspect}} must refer to a single aspect of
+                        |destination|.{{GPUTextureCopyView/texture}}.{{GPUTexture/[[format]]}}.
+                    - |destination|.{{GPUTextureCopyView/aspect}} must not refer to the depth aspect of
+                        |destination|.{{GPUTextureCopyView/texture}}.{{GPUTexture/[[format]]}}.
```

Comment:
These rules need to apply to writeTexture as well

Reply:
Ok I updated the copy commands and writeTexture as well to refer to the depth-formats table.